src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -89,20 +89,20 @@
 
 {% block dateinterval_widget %}
```

Comment:
we also have some missing whitespace control between the opening of the block and the if tag. Can you add it, like in time_widget ?